katsdpimageutils/render.py

```diff
@@ -130,31 +131,33 @@ def write_image(input_file, output_file, width=1024, height=768, dpi=DEFAULT_DPI
         Optional background color to use in the image plot window.
         Blanked pixels in the input FITS image will appear in this color.
     """
+    if not isinstance(input_file, fits.PrimaryHDU):
+        with fits.open(input_file) as hdus:
+            write_image(hdus[0], output_file, width, height, dpi, slices, caption, facecolor)
```

Comment:
@KimMcAlpine I realised this was broken because it doesn't return and falls through to the remaining code. I fixed it directly on master so that it wouldn't block me: https://github.com/ska-sa/katsdpimageutils/commit/f7d9ba65cbc02872f624f6d62aec4d747f36a313